packages/core/strapi/lib/services/entity-service/index.js

```diff
@@ -42,6 +43,7 @@ const updatePipeline = (data, context) => {
  */
 const createDefaultImplementation = ({ strapi, db, eventHub, entityValidator }) => ({
   uploadFiles,
+  validateModifiedRelations,
```

Comment:
What do you think of removing it from the entity service and just importing the function from the utils?
In a Oriented Object vision I think it belongs here but maybe not in a Functional Programming vision (which is the one we try to have)